crates/ruff_python_formatter/src/comments/format.rs

```diff
@@ -206,8 +206,15 @@ impl Format<PyFormatContext<'_>> for FormatDanglingComments<'_> {
             .iter()
             .filter(|comment| comment.is_unformatted())
         {
-            if first && comment.line_position().is_end_of_line() {
-                write!(f, [space(), space()])?;
+            if first {
+                match comment.line_position {
+                    CommentLinePosition::OwnLine => {
+                        write!(f, [hard_line_break()])?;
```

Comment:
I believe the reason for not using a `hard_line_break` mainly was that it may result in syntax errors if the enclosing node isn't parenthesized. But we can see how it goes